Modules/Core/Common/include/itkProcessObject.h

```diff
@@ -32,6 +32,7 @@
 #include "itkObjectFactory.h"
 #include "itkNumericTraits.h"
 #include "itkThreadSupport.h"
+#include "itkIntTypes.h"
```

Comment:
No problem, but I did notice that "itkObjectFactory.h" already indirectly includes "itkIntTypes.h", via itkObjectFactoryBase.h and then via itkCommonEnums.h:

https://github.com/InsightSoftwareConsortium/ITK/blob/5e726de40885e0f737dabcfa70e69324b4eed850/Modules/Core/Common/include/itkCommonEnums.h#L23 

While your proposed `#include "itkIntTypes.h"` is fine to me, I wonder if it has any effect.

Reply:
Yes, this seems to be a compiler issue. It addresses issues reported in #3506